Assets/Scripts/StateManager.cs

```diff
@@ -165,6 +167,9 @@ public void UpdateSites(bool forceRefresh)
                     }
                 }
             })
+            .Then(() => {
+                _wanManager.UpdateWans();
+            })
             .Catch(err => Debug.LogError($"Error updating sites/sitelinks: {err.Message}\n{err.StackTrace}"));
```

Comment:
If you can be bothered you can change this error message to say "Error updating entities" or something since it's not just sites and sitelinks anymore.